vue/official/list/src/components/MessageListItem.vue

```diff
@@ -6,7 +6,7 @@
         {{ message.fromName }}
         <span class="date">
           <ion-note>{{ message.date }}</ion-note>
-          <ion-icon :icon="chevronForward" size="small" v-if="isIos()"></ion-icon>
+          <ion-icon :aria-hidden="true" :icon="chevronForward" size="small" v-if="isIos()"></ion-icon>
```

Comment:
```suggestion
          <ion-icon aria-hidden="true" :icon="chevronForward" size="small" v-if="isIos()"></ion-icon>
```

`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.